net/connstats/stats.go

```diff
@@ -135,19 +135,19 @@ func (s *Statistics) updateVirtual(b []byte, receive bool) {
 // The src is always a Tailscale IP address, representing some remote peer.
 // The dst is a remote IP address and port that corresponds
 // with some physical peer backing the Tailscale IP address.
-func (s *Statistics) UpdateTxPhysical(src netip.Addr, dst netip.AddrPort, n int) {
-	s.updatePhysical(src, dst, n, false)
+func (s *Statistics) UpdateTxPhysical(src netip.Addr, dst netip.AddrPort, packets, n int) {
```

Comment:
Should we rename `n` to `bytes`? (here and below)

It seems like this is a fix to existing stats, so perhaps copy it into a separate PR that can be reviewed and merged sooner?

Reply:
Opened here: https://github.com/tailscale/tailscale/pull/13549